opencog/openpsi/README.md

```diff
@@ -42,21 +43,8 @@ are used).
 * The demand-update code is undocumented, confusing, and does not
   belong in this directory.
 
-
-### References
-* OpenCog Wiki - [OpenPsi](http://wiki.opencog.org/w/OpenPsi)
-* Hanson Robotics Wiki - [Emotion modelling](http://wiki.hansonrobotics.com/w/Emotion_modeling)
-* [MicroPsi publications](http://micropsi.com/publications/publications.html)
-* [MicroPsi source code]()
-* Joscha Bach's book -- [Principles of Synthetic Intelligence](http://wiki.humanobs.org/_media/public:events:agi-summerschool-2012:psi-oup-version-draft-jan-08.pdf) (2008 ed.)
-* Joscha Bach's book -- [Principles of Synthetic Intelligence](http://micropsi.com/publications/assets/Draft-MicroPsi-JBach-07-03-30.pdf) (2007 ed.)
-* Wikipedia - [Belief–desire–intention software model](https://en.wikipedia.org/wiki/Belief%E2%80%93desire%E2%80%93intention_software_model) has some
-  similarities with respect to demand, goals and actions, that is, has
-  the cognitive schematic `Belief + Intention ==> Desire, which is
-  similar to the Context + Procedure/Actions ==> Goal`which is used
-  here.
```

Comment:
The other URL's can be removed, but the wikipedia one seems to be fairly important, as it describes clearly the general concepts.